### DISCUSSION TITLE
posts/%E5%A5%87%E6%8A%80%E6%B7%AB%E5%B7%A7/%E9%97%AD%E5%8C%85%E5%AE%9E%E7%8E%B0%E7%B1%BB

### DISCUSSION THREAD

**Discussion by giscus** [King-sj/KBlog#2] [Q&A]
# posts/%E5%A5%87%E6%8A%80%E6%B7%AB%E5%B7%A7/%E9%97%AD%E5%8C%85%E5%AE%9E%E7%8E%B0%E7%B1%BB

闭包实现类 alt textalt text

http://localhost:8083/posts/%E5%A5%87%E6%8A%80%E6%B7%AB%E5%B7%A7/%E9%97%AD%E5%8C%85%E5%AE%9E%E7%8E%B0%E7%B1%BB.html

**Comment by King-sj** [comment]
test
